tests/test_spd_matrices.py

```diff
@@ -29,14 +29,14 @@ def setUp(self):
         self.metric_logeuclidean = SPDMetricLogEuclidean(n=self.n)
         self.n_samples = 4
 
-    @geomstats.tests.np_and_tf_only
+    @geomstats.tests.np_only
```

Comment:
Why doesn't it pass tf tests?

Reply:
shape mismatch? otherwise no idea, I think it should be fixed from the backend

Reply:
I'll do some tests of `gs.where`, `gs.linalg.det` etc and add a "#FIXME" as soon as it's figured out

Reply:
Yes, it relates to the vectorization issue... Would it be possible to fix it now? I know it's not fun, but otherwise, it's a step back in the code base as @ythanwerdas 's code was working before, and we should not take the habit of breaking existing unit tests if not really required?